src/cljs/athens/style.cljs

```diff
@@ -105,7 +153,34 @@
    :width    "100vw"})
 
 
-(stylefy/tag "body" base-styles)
+(stylefy/tag "html" base-styles)
+
+
+(stylefy/tag ":root" {:--link-color         (:link-color THEME-LIGHT)
+                      :--highlight-color    (:highlight-color THEME-LIGHT)
+                      :--warning-color      (:warning-color THEME-LIGHT)
+                      :--confirmation-color (:confirmation-color THEME-LIGHT)
+                      :--header-text-color  (:header-text-color THEME-LIGHT)
+                      :--body-text-color    (:body-text-color THEME-LIGHT)
+                      :--border-color       (:border-color THEME-LIGHT)
+                      :--background-minus-1 (:background-minus-1 THEME-LIGHT)
+                      :--background-minus-2 (:background-minus-2 THEME-LIGHT)
+                      :--background-color   (:background-color THEME-LIGHT)
+                      :--background-plus-1  (:background-plus-1 THEME-LIGHT)
+                      :--background-plus-2  (:background-plus-2 THEME-LIGHT)
```

Comment:
Use this function instead of hard-coding css vars

```
(defn remap-theme-keys
  "Maps theme keys to css variable keys."
  [theme]
  (reduce-kv
    (fn [m k v]
      (let [css-k (keyword (str "--" (symbol k)))]
        (assoc m css-k v)))
    {}
    theme))
```

Reply:
`reduce-kv` is another way to generate a map. Yesterday we did it with `apply hash-map` and `seq`